js/templates/components/moderatorCard.html

```diff
@@ -30,33 +30,37 @@
         </div>
         <div class="row">
           <% if (ob.valid) { %>
-            <div class="rowSm clamp2"><%=ob.moderatorInfo.description %></div>
+            <div class="rowTn clamp2"><%=ob.moderatorInfo.description %></div>
             <% if (ob.modLanguages && ob.modLanguages.length) { %>
-              <div class="txSm">
+              <div class="txSm rowTn">
                 <% if (ob.modLanguages.length > 1) {
                      print(ob.polyT('moderatorCard.languages', { lang: ob.modLanguages[0], smart_count: ob.modLanguages.length -1 }));
                    } else { %>
                   <%= ob.modLanguages[0] %>
                 <% } %>
               </div>
             <% } %>
-          </div>
-          <div class="flex snipKids gutterH tx5 detailsRow">
-            <div class="flexNoShrink TODO">
-              <%= ob.parseEmojis('👍') %> XX<% // placeholder for reputation %>
-            </div>
-            <div class="flexNoShrink">
-              <% var amount = ob.currencyMod.convertAndFormatCurrency(ob.moderatorInfo.fee.fixedFee.amount, ob.moderatorInfo.fee.fixedFee.currencyCode, ob.displayCurrency) %>
-              <%= ob.polyT(`moderatorCard.${ob.moderatorInfo.fee.feeType}`, { amount: amount, percentage: ob.moderatorInfo.fee.percentage }) %>
+            <div class="flex snipKids gutterH tx5 detailsRow">
+              <% if (ob.hasValidCurrency) { %>
+                <div class="flexNoShrink TODO">
```

Comment:
Can you comment on your usage of TODO? There's a comment for it in the stylesheet, but I'd like to know more clearly how you and @rmisio intend to use it.

Reply:
Sometimes we have features that were in the design but we can't complete because they're waiting on other work, but we want to make sure they're in the template so adding them won't require a lot of template and CSS  refactoring. 

There's some point at which things have been "TODO" for long enough they can be deleted, that would be a team discussion for each of them.